stubs/psutil/psutil/_common.pyi

```diff
@@ -106,23 +108,23 @@ class suser(NamedTuple):
 
 class sconn(NamedTuple):
     fd: int
-    family: Any
-    type: Any
+    family: AddressFamily
+    type: SocketKind
     laddr: str
     raddr: str
     status: str
     pid: int
 
 class snicaddr(NamedTuple):
-    family: Any
+    family: AddressFamily
     address: str
     netmask: str | None
     broadcast: str | None
     ptp: str | None
 
 class snicstats(NamedTuple):
     isup: bool
-    duplex: Any
+    duplex: int
```

Comment:
This is an enum to me:
```
>>> psutil.net_if_stats()
{'eth0': snicstats(isup=True, duplex=NicDuplex.NIC_DUPLEX_FULL, speed=100, mtu=1500), 'lo': snicstats(isup=True, duplex=NicDuplex.NIC_DUPLEX_UNKNOWN, speed=0, mtu=65536)}
>>> psutil.__version__
'5.8.0'
```

Reply:
It might be `int` or `IntEnum`: https://github.com/giampaolo/psutil/blob/master/psutil/_common.py#L130-L140 
The safest way is to assume `int`, which is a supertype of `IntEnum`.